{ 'stand_up_front_400': { 'head': [],
                          'left_arm': [ { 'duration': 1.0,
                                          'name': u'stand_up_front_1100',
                                          'positions': [ 0.0005,
                                                         -3.1358,
                                                         0.001,
                                                         3.118,
                                                         -1.8794,
                                                         1.528,
                                                         0.5535],
                                          'starttime': 4.003},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_1200',
                                          'positions': [ -2.1717,
                                                         -3.1418,
                                                         -0.0007,
                                                         3.1326,
                                                         -1.8784,
                                                         1.5346,
                                                         -1.1573],
                                          'starttime': 5.003},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_1300',
                                          'positions': [ -2.2113,
                                                         -1.5009,
                                                         0.0,
                                                         3.1323,
                                                         -1.8811,
                                                         1.5404,
                                                         -1.1305],
                                          'starttime': 6.003},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_1400',
                                          'positions': [ -0.0189,
                                                         -0.6914,
                                                         -0.002,
                                                         3.1295,
                                                         -0.0163,
                                                         1.5377,
                                                         -1.1454],
                                          'starttime': 7.003}],
                          'left_leg': [],
                          'right_arm': [ { 'duration': 1.0,
                                           'name': u'stand_up_front_1100',
                                           'positions': [ -0.0005,
                                                          3.1358,
                                                          -0.001,
                                                          -3.118,
                                                          1.8794,
                                                          -1.528,
                                                          -0.5535],
                                           'starttime': 0.0},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_1200',
                                           'positions': [ 2.1717,
                                                          3.1418,
                                                          0.0007,
                                                          -3.1326,
                                                          1.8784,
                                                          -1.5346,
                                                          1.1573],
                                           'starttime': 1.0},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_1300',
                                           'positions': [ 2.2113,
                                                          1.5009,
                                                          -0.0,
                                                          -3.1323,
                                                          1.8811,
                                                          -1.5404,
                                                          1.1305],
                                           'starttime': 2.0},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_1400',
                                           'positions': [ 0.0189,
                                                          0.6914,
                                                          0.002,
                                                          -3.1295,
                                                          0.0163,
                                                          -1.5377,
                                                          1.1454],
                                           'starttime': 3.0}],
                          'right_leg': [],
                          'torso': []}}
